src/enums/move-use-type.ts

```diff
@@ -0,0 +1,64 @@
+import { BattlerTagLapseType } from "#app/data/battler-tags";
+import { PostDancingMoveAbAttr } from "#app/data/abilities/ability";
```

Comment:
```suggestion
import type { BattlerTagLapseType } from "#app/data/battler-tags";
import type { PostDancingMoveAbAttr } from "#app/data/abilities/ability";
```
Not sure why Biome didn't do this already...